charts/consul/templates/server-statefulset.yaml

```diff
@@ -266,9 +284,12 @@ spec:
               {{- end }}
 
               {{ template "consul.extraconfig" }}
+              
+              WAN_ADDR="$(cat /consul/data/address.txt)"
 
               exec /usr/local/bin/docker-entrypoint.sh consul agent \
                 -advertise="${ADVERTISE_IP}" \
+                -hcl="external_addresses = [\"${WAN_ADDR}\"]" \
```

Comment:
This would be updated to include addresses from the helm value global.peering.service.serverExternalAddrs